src/bindings/python/flux/util.py

```diff
@@ -218,7 +218,7 @@ def __call__(self, main_func):
             exit_code = ex
         except Exception as ex:  # pylint: disable=broad-except
             exit_code = 1
-            # Prefer '{strerror}[: {filename}' error message over default
+            # Prefer '{strerror}: {filename}' error message over default
```

Comment:
@grondo you have a typo in the commit message :-)  "THere" -> "There"

Reply:
Oh my 😮